src/main/java/frc/robot/autochooser/CrossLine.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.autochooser;
+
+
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.RobotContainer;
+import frc.robot.commands.drivetrain.MoveDistance;
+import frc.robot.commands.ramp.ResetRamp;
+import frc.robot.subsystems.Ramp;
+import frc.robot.subsystems.SwerveDrivetrain;
+
+public class CrossLine extends ParallelCommandGroup {
+    double direction;
+    public CrossLine(SwerveDrivetrain drivetrain, Ramp ramp) {
+        if (RobotContainer.isRedAlliance() == true) {
```

Comment:
Can be simplified with a ternary operator or by eliminating == true and replacing == false with !
`direction = RobotContainer.isRedAlliance() ? -1.715 : 1.715`